### ISSUE TITLE
[FR] Mood tracker

### ISSUE THREAD

**Issue by OffifialMITX** [ramack/ActivityDiary#257]
Hey, I like this app very much, and I'd like it to include a mood tracker (like something as simple as a value from -10 to 10 which you can adjust in the notification, or maybe something more complex with multiple emotions) with a diagram that would show everything that's been recorded.

**Comment by ramack** [comment]
Yes, nice idea. I think this should be based on #113 while the data element attached to the.mood activity can be an enum for the different moods. -> we'd like to have icons assignable to the enum values for easy selection in the notification.
